### PR TITLE
Vault of magic additions

### PR DESCRIPTION
**Pull request by mshea**
If someone can run this in a local instance to make sure it works, that'd be great. I don't have a great setup for running a local instance to be sure it's cool.

### CODE REVIEW

**Comment by augustjohnson** [comment]
I resolved the conflict.

File: data/tome_of_beasts_3/vault_of_magic_text_dump.txt
Comment (Sturlen) [inline data/tome_of_beasts_3/vault_of_magic_text_dump.txt:1]: The source file should probably be contained within the vault of magic folder, not tome of beasts.

Comment (mshea) [reply data/tome_of_beasts_3/vault_of_magic_text_dump.txt:1]: In my local branch it's i a "vault_of_magic" directory underneath the data directory. Is that not how it ended up?

Comment (Sturlen) [reply data/tome_of_beasts_3/vault_of_magic_text_dump.txt:1]: if you look under the Files Changed on Github, you can see that the text file appears to be in the wrong folder:
image
There also seem to be a lot of unrelated files that have been changed too, not sure of the cause.

**Pull request closed without merging**